pllug-presentation-system/presentation-data/concretepresentationelementfactory.cpp

```diff
@@ -0,0 +1,30 @@
+#include "concretepresentationelementfactory.h"
+
+#include <QString>
+
+#include "presentationelement.h"
+#include "header.h"
+#include "paragraph.h"
+#include "separator.h"
+
+std::unique_ptr<PresentationElement> ConcretePresentationElementFactory::create(const QString &html) const
```

Comment:
And it is better to add prefix that will tell - in what way this factory implementation is specific.